README.md

```diff
@@ -118,6 +118,66 @@ function ExampleWrapper({
 
 [Try it on Code Sandbox](https://codesandbox.io/s/x70ly749rq)
 
+## Advanced usage
+
+Some use-cases need the cached rows to be cleared for example a sorted list. Otherwise the InfiniteLoader won't call the function passed in `loadMoreItems` prop. This can be achieved by calling the `InfiniteLoader` component's method `resetloadMoreItemsCache`.
+
+```jsx
+function ExampleWrapper({
+  hasNextPage,
+  isNextPageLoading,
+  items,
+  loadNextPage,
+  sort,
```

Comment:
nit: `sortOrder` or `sortDirection`? Something that sounds less like a function?

Reply:
You are right!